guides/common/modules/proc_installing-puppet-agent-during-host-provisioning.adoc

```diff
@@ -39,16 +39,27 @@ endif::[]
 <%= snippet 'puppet_setup' %>
 ----
 ifndef::katello,orcharhino,satellite[]
-. Enable the official Puppet repository using a host parameter in global parameters, a host group, or for a single host.
+. Enable the Puppet agent from the official Puppet repository using one of the following host parameters in global parameters, a host group, or for a single host:
+
+* Add a host parameter named `enable-puppetlabs-repo` for the latest stable Puppet release in the unversioned repository.
+* Add a host parameter named `enable-official-puppet7-repo` for the Puppet 7 release repository.
+* Add a host parameter named `enable-puppetlabs-puppet6-repo` for the Puppet 6 release repository.
+
 +
-Add a host parameter named `enable-puppetlabs-repo` for the latest Puppet 7 or `enable-puppetlabs-puppet6-repo` for Puppet 6, select the `boolean` type, and set the value to `true`.
+Select the `boolean` type and set the value to `true`.
 endif::[]
+ifdef::katello[]
 . Enable Puppet agent using a host parameter in global parameters, a host group, or for a single host.
 +
-ifndef::satellite[]
 Add a host parameter named `enable-puppet7` for Puppet 7 or `enable-puppet6` for Puppet 6, select the `boolean` type and set the value to `true`.
+ifdef::orcharhino[]
+. Enable Puppet agent using a host parameter in global parameters, a host group, or for a single host.
++
+Add a host parameter named `enable-puppet6` for Puppet 6, select the `boolean` type and set the value to `true`.
```

Comment:
Any reason to use Puppet 6 here? @maximiliankolb?

I'll stress that Puppet 6 is about to be end of life so I'd urge you to get to Puppet 7.

Reply:
Waiting with another push, until we clarify this.